templates/mwe.md

```diff
@@ -58,21 +50,23 @@ h2 : a.fst < b.snd
 ### Good example:
 
 ```lean
-def blah : Type := ℕ × ℕ
-
-example (a b : blah) (h : a.fst < b.fst) (h2 : a.fst < b.snd) : false :=
-begin
-
-end
-/-
-a b : blah,
-h : a.fst < b.fst,
-h2 : a.fst < b.snd
-⊢ false
--/
+def blah : Type := Nat × Nat
+
+example (a b : blah) (h : a.fst < b.fst) (h2 : a.fst < b.snd) : False := by
+  /-
+  a b : blah,
+  h : a.fst < b.fst,
+  h2 : a.fst < b.snd
+  ⊢ False
+  -/
+  done
 ```
 
-Tip: If you are using [mathlib](https://github.com/leanprover-community/mathlib) and have `import tactic` in your file, there's a tactic called [`extract_goal`](https://leanprover-community.github.io/mathlib_docs/tactics.html#extract_goal) that can help you format the current goal as a stand-alone example. You can remove extraneous variables and hypotheses from the output of `extract_goal` to further minimize your example.
+<!---
+TODO: update link to `extract_goal` once the lean4-equivalent of `tactics.html` exists.
+-->
+
+Tip: If you are using [mathlib](https://github.com/leanprover-community/mathlib4) (e.g. with `import Mathlib`), there's a tactic called [`extract_goal`](https://leanprover-community.github.io/mathlib_docs/tactics.html#extract_goal) that can help you format the current goal as a stand-alone example. You can remove extraneous variables and hypotheses from the output of `extract_goal` to further minimize your example.
```

Comment:
Please link to https://leanprover-community.github.io/mathlib4_docs/Mathlib/Tactic/ExtractGoal.html

Reply:
done